README.md

```diff
@@ -1,5 +1,41 @@
-To see the basic concept working:
-1. run a copy of [montagu-db](https://github.com/vimc/montagu-db): `./montagu-db/scripts/start.sh master 5432`
-2. you'll also need to add a user: `docker exec -it db psql -U postgres -d montagu -c "CREATE USER montagu WITH PASSWORD 'montagu'; GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA public TO montagu;"`
-3. create the sql functions: `psql -h localhost -U montagu -d montagu -f create_mock_table.sql -f create_mock_schema.sql` (enter password "montagu" when prompted)
-4. run `create_mocks.R` to generate a file called `generated.R` containing functions for creating mock tables
+# Fakerbase
+[![codecov](https://codecov.io/gh/reside-ic/fakerbase/branch/master/graph/badge.svg?token=PSbEOyI1yi)](https://codecov.io/gh/reside-ic/fakerbase)
+[![Project Status: WIP – Initial development is in progress, but there has not yet been a stable, usable release suitable for the public.](https://www.repostatus.org/badges/latest/wip.svg)](https://www.repostatus.org/#wip)
+![R-CMD-check](https://github.com/reside-ic/fakerbase/actions/workflows/R-CMD-check.yml/badge.svg)
+
+Use case: unit testing code that manipulates data frames obtained by querying a SQL database. Auto-generate functions for 
+creating in memory data frames that mirror the structure of the database, for easy mocking of the tables. 
+
+## Usage
+
+1. Create a db connection
+
+```{r}
+    con <- DBI::dbConnect(RPostgres::Postgres(),
```

Comment:
It would be good to get a self-contained example. We might use something like the classic [Northwind schema](https://docs.yugabyte.com/latest/sample-data/northwind/) (I think that @r-ash pointed me at this) and add a helper in the tests to create that. Could be done now or later